manifests/install.pp

```diff
@@ -1,15 +1,15 @@
 # Install the foreman proxy
 class foreman_proxy::install {
   if ! $foreman_proxy::custom_repo {
-    foreman::repos { 'foreman_proxy':
+    class { '::foreman::repo':
```

Comment:
I now see that we do pull in EPEL and SCL as well. I'm also worried about duplicate resource definitions since puppet-foreman also calls it as ```class { '::foreman::repo': }```.